### PR TITLE
UI: MemberChip Abstraction and Permutations for Elections

### PR DESCRIPTION
**Pull request by brandonfancher**
Creates a `GenericMemberChip` component that is used by three new specific member chip components/permutations:

* `MemberChip`
* `VotingMemberChip`
* `WinningMemberChip`

These can be seen in action on the members pages and in a new page: `/election/components`.

image

image

### CODE REVIEW

File: packages/webapp/src/_app/ui/generic-member-chip.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    >
+        <div className="flex space-x-2.5">
+            <MemberImage
+                member={member}
+                onClick={onClickProfileImage || onClickChip}
+            />
+            {contentComponent}
+        </div>
+        {actionComponent}
+    </div>
```
Comment (cool-ant) [inline packages/webapp/src/_app/ui/generic-member-chip.tsx:32]: Wow. Love this organization

File: packages/webapp/src/members/components/member-chip.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                />
+                <p className="text-sm tracking-tight leading-none p-t-px group-hover:underline">
+                    {assetToString(member.auctionData.price, 2)} (#
+                    {member.assetData?.templateMint})
+                </p>
+            </div>
+        );
+    }
+
+    return <></>;
```
Comment (cool-ant) [inline packages/webapp/src/members/components/member-chip.tsx:121]: Just a preference, but I'm a fan of having the intended code be unwrapped/primary. So
if (!member.auctionData) return <></>
//rest of code unrwapped

**Review by cool-ant — APPROVED** [review]

File: packages/webapp/src/members/components/member-chip.tsx (hunk 2 above)
Comment (brandonfancher) [reply packages/webapp/src/members/components/member-chip.tsx:121]: Actually, that's usually my preference too. And less nesting. 👍

Comment (brandonfancher) [reply packages/webapp/src/members/components/member-chip.tsx]: Made that adjustment. Thanks!

**Comment by brandonfancher** [comment]
A couple more UI tweaks: 
image

**Pull request merged**